api/src/LinkyLink/Helpers/UserAuth.cs

```diff
@@ -23,39 +23,19 @@ public UserInfo GetUserAccountInfo()
         {
             if (_contextAccessor.HttpContext.User.Identity.IsAuthenticated)
             {
-                var socialIdentities = _contextAccessor.HttpContext.User.Identities
-                    .Where(id => !id.AuthenticationType.Equals("WebJobsAuthLevel", StringComparison.InvariantCultureIgnoreCase));
+                var identities = _contextAccessor.HttpContext.User.Identities;
 
-                if (socialIdentities.Any())
+                if (identities.Any())
                 {
-                    var provider = _contextAccessor.HttpContext.Request.Headers["X-MS-CLIENT-PRINCIPAL-IDP"].DefaultIfEmpty(string.Empty).FirstOrDefault();
-                    var primaryIdentity = socialIdentities.First();
-                    var email = primaryIdentity.Claims.SingleOrDefault(c => c.Type == ClaimTypes.Email).Value;
-                    var userInfo = new UserInfo(provider, HashString(email));
+                    var primaryIdentity = identities.First();
+                    var email = primaryIdentity.Claims.SingleOrDefault(c => c.Type == "emails").Value;
+                    var userInfo = new UserInfo("ADB2C", email);
```

Comment:
As we have changed this to return email address, should we rename the UserInfo.HashedID property to UserInfo.UserID or UserInfo.Email

Reply:
Going to keep as is for the moment.